src/components/Card/Card.scss

```diff
@@ -29,6 +29,8 @@
 
 .Section {
   padding: spacing();
+  border-bottom-left-radius: border-radius();
```

Comment:
If a `subdued` section is the first section within a `Card`, you could end up with `top left/right` "corner-bleeding".

Reply:
I switched this up a bit to target first and last child and only change the border radius on those. What do you think?

Reply:
Yep - this is essentially the same solution I used in a similar component. 🚢

Reply:
Nice. Would `.Card > :first-child` / `.Card > :last-child` help protect against cases where people put not-a-section as the first / last item (e.g. there's only a Card.Header for some weird reason) or is that not considered valid ?

Reply:
The issue is only apparent when the child element has _something_ that could bleed out. In the case of a `Section` - its actually specifically a `Section + subdued`, as it is the darkened `background-color` that is "bleeding out".

If there are other child elements that could be `subdued`, we should cast a wide enough net to catch those as well... but I don't _think_ a `Header` or `Footer` can be `subdued` _(I could be wrong)_. Otherwise, if its just any arbitrary children that the consumer might be passing, then I'd lean towards the onus being on the consumer to style those children to suppress the border-radius bleed.